pkg/ctrl/switchprofile/profile_celestica_ds5000.go

```diff
@@ -0,0 +1,136 @@
+			"E1/40": {NOSName: "1/40", BaseNOSName: "Ethernet312", Label: "40", Profile: "OSFP-1x800G"},
+			"E1/41": {NOSName: "1/41", BaseNOSName: "Ethernet320", Label: "41", Profile: "OSFP-1x800G"},
+			"E1/42": {NOSName: "1/42", BaseNOSName: "Ethernet328", Label: "42", Profile: "OSFP-1x800G"},
+			"E1/43": {NOSName: "1/43", BaseNOSName: "Ethernet336", Label: "43", Profile: "OSFP-1x800G"},
+			"E1/44": {NOSName: "1/44", BaseNOSName: "Ethernet344", Label: "44", Profile: "OSFP-1x800G"},
+			"E1/45": {NOSName: "1/45", BaseNOSName: "Ethernet352", Label: "45", Profile: "OSFP-1x800G"},
+			"E1/46": {NOSName: "1/46", BaseNOSName: "Ethernet360", Label: "46", Profile: "OSFP-1x800G"},
+			"E1/47": {NOSName: "1/47", BaseNOSName: "Ethernet368", Label: "47", Profile: "OSFP-1x800G"},
+			"E1/48": {NOSName: "1/48", BaseNOSName: "Ethernet376", Label: "48", Profile: "OSFP-1x800G"},
+			"E1/49": {NOSName: "1/49", BaseNOSName: "Ethernet384", Label: "49", Profile: "OSFP-1x800G"},
+			"E1/50": {NOSName: "1/50", BaseNOSName: "Ethernet392", Label: "50", Profile: "OSFP-1x800G"},
+			"E1/51": {NOSName: "1/51", BaseNOSName: "Ethernet400", Label: "51", Profile: "OSFP-1x800G"},
+			"E1/52": {NOSName: "1/52", BaseNOSName: "Ethernet408", Label: "52", Profile: "OSFP-1x800G"},
+			"E1/53": {NOSName: "1/53", BaseNOSName: "Ethernet416", Label: "53", Profile: "OSFP-1x800G"},
+			"E1/54": {NOSName: "1/54", BaseNOSName: "Ethernet424", Label: "54", Profile: "OSFP-1x800G"},
+			"E1/55": {NOSName: "1/55", BaseNOSName: "Ethernet432", Label: "55", Profile: "OSFP-1x800G"},
+			"E1/56": {NOSName: "1/56", BaseNOSName: "Ethernet440", Label: "56", Profile: "OSFP-1x800G"},
+			"E1/57": {NOSName: "1/57", BaseNOSName: "Ethernet448", Label: "57", Profile: "OSFP-1x800G"},
+			"E1/58": {NOSName: "1/58", BaseNOSName: "Ethernet456", Label: "58", Profile: "OSFP-1x800G"},
+			"E1/59": {NOSName: "1/59", BaseNOSName: "Ethernet464", Label: "59", Profile: "OSFP-1x800G"},
+			"E1/60": {NOSName: "1/60", BaseNOSName: "Ethernet472", Label: "60", Profile: "OSFP-1x800G"},
+			"E1/61": {NOSName: "1/61", BaseNOSName: "Ethernet480", Label: "61", Profile: "OSFP-1x800G"},
+			"E1/62": {NOSName: "1/62", BaseNOSName: "Ethernet488", Label: "62", Profile: "OSFP-1x800G"},
+			"E1/63": {NOSName: "1/63", BaseNOSName: "Ethernet496", Label: "63", Profile: "OSFP-1x800G"},
+			"E1/64": {NOSName: "1/64", BaseNOSName: "Ethernet504", Label: "64", Profile: "OSFP-1x800G"}, // 64x OSFP-1x800G
+			"E1/65": {NOSName: "Ethernet512", Label: "65", Profile: "SFP28-25G"},                        // 1x SFP-28-25G
+			"E1/66": {NOSName: "Ethernet513", Label: "66", Profile: "SFP28-25G"},                        // 1x SFP-28-25G
+		},
+		PortProfiles: map[string]wiringapi.SwitchProfilePortProfile{
+			"SFP28-25G": {
+				Speed: &wiringapi.SwitchProfilePortProfileSpeed{
+					Default:   "25G",
+					Supported: []string{"1G", "10G", "25G"},
+				},
+			},
+			"OSFP-1x800G": {
```

Comment:
I think it should be named OSFP-800G for consistency, as 1x is just a configuration

Reply:
Agreed. I will change it.